src/main/java/com/scalar/database/transaction/consensuscommit/PartitionedMutations.java

```diff
@@ -12,11 +12,18 @@
 import javax.annotation.Nonnull;
 import javax.annotation.concurrent.Immutable;
 
-/** */
+/**
+ * A class for partitioning mutations based on their partition key
```

Comment:
I don't think we need `A class` because it's obvious. (it applies to other docs.)
It should be more like 'Mutations partitioned by their partition key`.

Reply:
Thank you! Done!